spec/requests/profile_spec.rb

```diff
@@ -0,0 +1,164 @@
+# frozen_string_literal: true
+
+describe "Profile", type: :request do
+  describe "#index" do
+    context "when user is not logged in" do
+      before { get profile_index_path, params: {uid: "user-id"} }
+      it { expect(response).to redirect_to(new_user_session_path) }
+    end
+
+    context "when user is logged in" do
+      let(:user1) { create(:user1) }
+      before { sign_in user1 }
+      it "renders index template" do
+        get profile_index_path, params: {uid: user1.uid}
+        expect(response).to render_template(:index)
```

Comment:
```suggestion
      it "renders page" do
        get profile_index_path, params: {uid: user1.uid}
        expect(response).to be_successful
```

`render_template` is only available with `rails-controller-test` gem so let's just change this to `be_successful`

Reply:
I agree. I should have spent a bit more time looking at the `rails-controller-test` gem and what needed to be changed to remove it. I really just moved the controller test to the requests folder and made enough tweaks to get the tests to run/pass. I will make sure I am more attentive to changes in future PRs.